cpp/src/robotick/framework/data/TelemetryServer.cpp

```
@@ -0,0 +1,348 @@
+				}
+
+				return false;
+			});
+	}
+
+	void TelemetryServer::stop()
+	{
+		web_server.stop();
+	}
+
+	static bool type_already_emitted(const nlohmann::ordered_json& layout_json, const char* name)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)